modules/ROOT/pages/mule-deployment-model.adoc

```diff
@@ -0,0 +1,26 @@
+= Mule App Deployment Model
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, esb, deploy, architecture
+
+A Mule project for deployment is a Mule application. A Mule runtime instance supports multiple applications running independently within the same Mule container. This allows additional advantages such as:
+
+* Running independently multiple applications side-by-side. If you have to build an application with some complexity, you can divide it by creating several apps with their specific logic and then deploy all the apps within the same Mule container.
+* Clear boundaries for running operations on what a Mule application is.
+* Mule can monitor your applications and will reload configuration changes automatically.
+* Applications can depend on different library versions, even if these would conflict before.
+* Multiple versions of an application can run within the same Mule container.
+
+Every application is deployed together within a *domain*. By default, your application references the default domain, and no further configuration is required. You can see that it gets deployed together with the app in the console:
+
+image::deploy-domain.png[deploy+domain]
+
+If you're deploying multiple applications to the same place, and those applications could share the same resources, then you can create a common *domain* where you can define common configurations that can then be referenced by multiple projects. This allows you to, for example, expose different services in different projects through the same HTTP host and port and be able to deploy everything without any conflicts. xref:shared-resources.adoc[Read More].
```

Comment:
I suggest shorter sentences:

"This allows you to expose different services in different projects through the same HTTP host and port without causing any conflicts.

Reply:
Updated.